src/CMakeLists.txt

```diff
@@ -199,6 +199,12 @@ add_library(parthenon
   parthenon_mpi.hpp
 )
 
+add_library(parthenon "${parthenon_sources}")
+
+if (BUILD_SHARED_LIBS)
+  set_target_properties(parthenon PROPERTIES SOVERSION 1)
```

Comment:
We should probably have this derive from the project version which I'm finally setting in this PR: https://github.com/lanl/parthenon/pull/249

So maybe set this to `${parthenon_VERSION}`?

If you don't want to be blocked on that (but I'll try to get it merged shortly), you can set the project version to `0.0.1` or something.

Also, you shouldn't need to check `BUILD_SHARED_LIBS`. I think you can safely set this unconditionally.

Reply:
Looks like your PR #249 got merged in yesterday, I'm using `${parthenon_VERSION}` now, thanks.

I don't run into any issues removing the `BUILD_SHARED_LIBS` check here when compiling a static library